pages/volunteer/signup.tsx

```diff
@@ -183,36 +234,30 @@ export default function VolunteerInfo({
             headers: { "Content-Type": "application/json" },
             body: JSON.stringify(userData),
         };
-        const response = await fetch("api/user", request);
-        const updatedUserData = await response.json();
-        return updatedUserData;
+        const response = await fetch("/api/user", request);
+        if (response.status === 200) {
```

Comment:
I think this would be safer.
```suggestion
        if (response.ok) {
```

Reply:
Sure :)